src/test/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRowMapperTest.java

```diff
@@ -1,19 +1,19 @@
 package gov.usgs.wma.waterdata.groundwater;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Just a note that your IDE may have been too helpful here, depending on whether or not you like to see each individual import.

Reply:
I believe if you run this through Codacy it will flag this as we have it set up to enforce separate imports rather than the use of '*'. Personally I like the separate imports because it makes it clear to a future reader of the code where a function, class, etc are imported from.